### ISSUE TITLE
What is the final version of the library?

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#1863]
**Reported by beutlich on 16 Dec 2015 22:59 UTC**
Since 3cfd14d5f1894be8fb17f98c47b637368440f4cd the following note
```
// IMPORTANT NOTE
// This parameter may be removed in the final version of the library
// for consistency reasons with respect to the single phase
// electromagnetic converter, where the orientation shall NOT be
// implemented in the final version
```
was added to MultiPhaseElectroMagneticConverter.

Please revise that note or check if the orientation parameter can be removed as intended.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/1863

**Comment by modelica-trac-importer** [comment]
**Comment by ahaumer on 19 Dec 2015 21:16 UTC**
resolved by 811c96eb7a71c641c5896f9b27bbf34016f1d097

**Comment by modelica-trac-importer** [comment]
**Modified by ahaumer on 19 Dec 2015 21:16 UTC**
